### DISCUSSION TITLE
Desenvolvedores do Governo

### DISCUSSION THREAD

**Discussion by zemariobandeira** [frontendbr/forum#1205] [Discussão]
### Comunidade de Desenvolvedores do Governo
Olá pessoal, tudo bom?
Então, eu atualmente trabalho como desenvolvedor para uma instituição pública e senti a necessidade de termos uma comunidade focada em desenvolvedores que atuam em órgãos públicos e afins.
Pensando nisso, me reuni com alguns devs em grupos do telegram para pensar ações com o objetivo de engajar a comunidade de devs à formar um ecossistema de troca de conhecimento técnico entre desenvolvedores do governo.

Se você trabalha em alguma instituição pública e deseja colaborar com esse projeto, informe o seu usuário do telegram que nós iremos adicioná-lo ao grupo DevsGovBR.

Sua participação será fundamental para que possamos mudar a cultura trivial dos ambientes de desenvolvimento do governo a fim de fomentar a inovação nos setores públicos do país.

Desde já, agradeço a todos pelo apoio!

**Comment by didiraja** [comment]
Boa iniciativa, @mariodkr. Mas não seria interessante criar um repositório por aqui e manter mais acessível?

**Comment by zemariobandeira** [comment]
@didiraja , pode ser... é que, existem algumas observações a serem consideradas, com relação a sigilo de informações sensíveis e afins... seria bom fazer um brainstorm com os profissionais da área pra ver se um repositório aqui no github atenderia bem a proposta sem ferir nenhum princípio ético, etc. Por isso a criação do grupo no telegram. É mais para promover a troca de ideias para a comunidade.

De qualquer forma, segue o link do grupo, para os interessados: https://t.me/joinchat/HFh0CAuJJyvSfm1IzbDJJw

Obrigado pela colaboração, @didiraja 
Toda sugestão será bem vinda.

**Comment by fakenickels** [comment]
Essa é uma ideia muito boa!
Meu sonho era ver coisas do governo virando open-source e usando a própria comunidade como auditoria e correção de falhas e etc. Ia acabar com muita conspiração por aí.

**Comment by fakenickels** [comment]
O que vc acha de ter um Discord server pra isso @mariodkr? Acho que ficaria bem legal pra compartilhar código e etc

**Comment by zemariobandeira** [comment]
@grsabreu , também parece ser uma ótima plataforma para estabelecer a comunidade.
Cola lá no grupo e sugere pro pessoal...

**Comment by LuanFreitas** [comment]
Baita iniciativa fera, parabéns!
Nunca trabalhei pro Gov mas a gente que tá de fora escuta muito sobre as coisas triviais que acontecem e em alguns serviços até percebemos isso. Tomara que dê muito certo!

**Reply by LESS14** [reply]
Sei que é uma discussão antiga, mas já trabalhei o bastante com programação(Uma vez até mesmo para a prefeitura da minha cidade) para ter uma opinião formada que creio que muitos vão se identificar. Eu arriscaria dizer que infelizmente, 90% dos serviços do Governo Federal sofrem com falhas, sobrecargas, problemas de infraestrutura, travamentos entre outros, muito disso vem do fato de que trabalhar no Governo, não apenas como desenvolvedor, mas como praticamente todo servidor público, você é obrigado a passar por 1 tonelada de processos burocráticos antes de implementar qualquer coisa, e pros desenvolvedores, é ainda pior, pois eles não têm a liberdade de implementar novas features no programa/serviço que estão desenvolvendo de um jeito apropriado por culpa dos milhares de processos burocráticos. E eu acho que isso é um problema muito grave no Governo Brasileiro. É bem triste de ver que um país tão rico como o Brasil seja talvez um dos países menos desenvolvidos tecnologicamente dentro do próprio Governo. Muitos sistemas do Governo se baseiam em "Gambiarras" absurdas no código tornando aquilo insustentável à longo prazo. Realmente espero que isso mude algum dia e grandes reformas sejam feitas em relação aos sistemas digitais do Governo e o modo como eles tratam e lidam com os Desenvolvedores com tanta burocracia.

**Comment by daniloarodrigues-zz** [comment]
Boa iniciativa.
